robotiq_cpp_control/launch/rc_gripper.launch

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) <2016>, <Matthew Horn>
```

Comment:
Launch files don't typically require a license as they are not considered software.